src/components/_project/testimonial.module.scss

```diff
@@ -1,82 +1,111 @@
 @import '../../base/module';
 
 .Testimonial {
-    padding-top: 0;
-}
-.bannerWrap{
+  padding: 0;
+  margin-bottom: 1.5625rem;
+  background-color: $color-white;
+
+  .bannerWrap {
     width: calc(100% + 40px);
-    margin: 0 -20px 95px;
+    margin: 0 -1.25rem 1rem;
```

Comment:
Margins because they aren't type related can stay as pixels if that's easier but this works fine, don't change it lol.